### DISCUSSION TITLE
How do I get a source map for large first-party JavaScript in Next.js?

### DISCUSSION THREAD

**Discussion by Elindo586** [vercel/next.js#64303] [Help] [answered]
### Summary

I am working on some issues shown from Google Lighthouse.

One of the issues is:
```
Missing source maps for large first-party JavaScript
Source maps translate minified code to the original source code. This helps developers debug in production. In addition, Lighthouse is able to provide further insights. Consider deploying source maps to take advantage of these benefits. [Learn more about source maps](https://developer.chrome.com/docs/devtools/javascript/source-maps/?utm_source=lighthouse&utm_medium=devtools).
```
![image](https://github.com/vercel/next.js/assets/93617433/f0b7d621-c788-4503-b97a-b84f1c04bca3)

^^ How do I go about to include this in a Next.js project?

**Comment by icyJoseph** [comment, marked as answer]
- https://nextjs.org/docs/app/api-reference/next-config-js/productionBrowserSourceMaps

The path to solve Lighthouse issues is but a fun one...

**Reply by Elindo586** [reply]
Thank you much!
